# https://chromium.googlesource.com/chromium/src/+/a5252b20a67b9a6882090d963fbd4dc168dd3436%5E%21/
# Remove restrictions for enabling Widevine support as the libwidevinecdm.so
# component should be able to be installed separately from the browser.

--- a/third_party/widevine/cdm/BUILD.gn
+++ b/third_party/widevine/cdm/BUILD.gn
@@ -8,12 +8,6 @@ import("//media/cdm/library_cdm/cdm_path
 import("//media/media_options.gni")
 import("//third_party/widevine/cdm/widevine.gni")
 
-# Internal Cast builds set enable_widevine=true to bring in Widevine support.
-# TODO(xhwang): Support component updated CDM on other platforms and remove this
-# assert.
-assert(!enable_widevine || is_win || is_mac || is_chromecast,
-       "Component updated CDM only supported on Windows and Mac for now.")
-
 widevine_arch = current_cpu
 if (widevine_arch == "x86") {
   widevine_arch = "ia32"
